### PR TITLE
Added ActionButton.

### PR DESCRIPTION
**Pull request by millevlada**
Added disabled reason for PopupMenu2 options, so it can better reuse IUIAction. Also added options to use text and hint as plain text. Fixed z-index for popup menu given inside floating panels.

### CODE REVIEW

File: to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	@Override
+	public void createContent() throws Exception {
+		super.createContent();
+		if(m_actions.isEmpty()) {
+			return;
+		}
+		HR hr = new HR();
+		hr.addCssClass("act-rule");
+		add(hr);
+		NodeBase actionButton = Icon.faEllipsisV.createNode();
```
Comment (fjalvingh) [inline to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java]: If the button is disabled the menu needs to be disabled too.

File: to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+	private final List<IUIAction<T>> m_actions = new ArrayList<>();
+
+	private final T m_instance;
+
+	public ActionButton(T instance, IUIAction<T> action) throws Exception {
+		super(action);
+		m_instance = instance;
+	}
+
+	public ActionButton<T> addAction(IUIAction<T> action) {
```
Comment (fjalvingh) [inline to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java]: Why would all instances need to be a "T"? I think both the primary action and the additional actions could have different instances?

File: to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java (hunk 1 above)
Comment (fjalvingh) [reply to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java]: On a button of this type the icon is not a ... but a v (down triangle)

**Review by fjalvingh — CHANGES_REQUESTED** [review]

File: to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java (hunk 1 above)
Comment (millevlada) [reply to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java]: ok, fixed

File: to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java (hunk 2 above)
Comment (millevlada) [reply to.etc.domui/src/main/java/to/etc/domui/component/buttons/ActionButton.java]: fixed

**Review by fjalvingh — APPROVED** [review]

**Pull request merged**